src/app/components/LayerPicker/index.tsx

```diff
@@ -137,12 +136,6 @@ const LayerPickerContent: FC<LayerPickerContentProps> = ({ isOutOfDate, onClose,
                 </TabletBackButton>
               )}
           </div>
-          <MobileNetworkButton
```

Comment:
I expect this was intentional design for visual balance

Reply:
Maybe so. I spelled out my rationale in the PR description, and asked @juresobocan for input.